iceoryx_hoofs/source/posix_wrapper/shared_memory_object/memory_map.cpp

```diff
@@ -60,11 +61,11 @@ cxx::expected<MemoryMap, MemoryMapError> MemoryMapBuilder::create() noexcept
 
     constexpr uint64_t FLAGS_BIT_SIZE = 32U;
     auto flags = std::cerr.flags();
-    std::cerr << "Unable to map memory with the following properties [ baseAddressHint = " << std::hex
-              << m_baseAddressHint << ", length = " << std::dec << m_length << ", fileDescriptor = " << m_fileDescriptor
-              << ", access mode = " << asStringLiteral(m_accessMode)
-              << ", flags = " << std::bitset<FLAGS_BIT_SIZE>(static_cast<uint32_t>(flags)) << ", offset = " << std::hex
-              << m_offset << std::dec << " ]" << std::endl;
+    IOX_LOG(ERROR) << "Unable to map memory with the following properties [ baseAddressHint = " << std::hex
```

Comment:
Getting an incorrect implicit conversion here:

```
/home/runner/work/iceoryx/iceoryx/iceoryx_hoofs/source/posix_wrapper/shared_memory_object/memory_map.cpp:64:100: error: implicit conversion 'std::ios_base &(*)(std::ios_base &)' -> bool [readability-implicit-bool-conversion,-warnings-as-errors]
Suppressed 57434 warnings (57140 in non-user code, 284 NOLINT, 10 with check filters).
    IOX_LOG(ERROR) << "Unable to map memory with the following properties [ baseAddressHint = " << std::hex
                                                                                                   ^
                                                                                                   (        != nullptr)
```

Anyone have any suggestions on the best way to deal with this ?

Reply:
@ithier you have to use `iox::log::hex(m_offset)`